jf_agent/main.py

```diff
@@ -150,10 +138,9 @@ def main():
             if config.run_mode_includes_download:
                 download_data_status = download_data(
                     config,
+                    creds,
                     jellyfish_endpoint_info.jira_info,
-                    jellyfish_endpoint_info.git_instance_info,
-                    jira_connection,
-                    git_connection,
+                    jellyfish_endpoint_info.git_instance_info,  # todo
```

Comment:
What's the TODO here for? Has this been addressed?

Reply:
Yup, it's been addressed, it's just an artifact.